meta/sai_meta.cpp

```diff
@@ -1705,6 +1725,151 @@ sai_status_t meta_generic_validation_create(
     return SAI_STATUS_SUCCESS;
 }
 
+bool meta_is_object_in_default_state(
+        _In_ sai_object_id_t oid)
+{
+    SWSS_LOG_ENTER();
+
+    if (oid == SAI_NULL_OBJECT_ID)
+        SWSS_LOG_THROW("not expected NULL object id");
+
+    //sai_object_type_t object_type = sai_object_type_query(oid);
+
+    // TODO for each oid - check reference count if it's safe to remove
```

Comment:
is this still TODO?

Reply:
forgot to remove that, this is checked in line 1831 and 1849